packages/file_selector/file_selector_platform_interface/lib/src/types/x_type_group/x_type_group.dart

```diff
@@ -26,7 +26,7 @@ class XTypeGroup {
   final List<String>? mimeTypes;
 
   /// The UTIs for this group
-  final List<String>? macUTIs;
+  List<String>? macUTIs;
```

Comment:
This should stay final; there shouldn't be a setter for it. I forgot this was a conceptually immutable class when I wrote that example of forwarding constructs in the issue.

Reply:
(Filed https://github.com/flutter/flutter/issues/111906 since this change shouldn't have been possible in the first place.)